src/components/UNSTABLE_Pagination/_index.scss

```diff
@@ -0,0 +1,13 @@
+////
+/// Unstable pagination component.
+/// @group unstable-pagination
+/// @copyright IBM Security 2019
+////
+
+@import '../Component/mixins';
+
+@import 'mixins';
+
+@include security--component($name: unstable-pagination) {
+  @include unstable-pagination;
```

Comment:
If you don't mind, can we remove the `mixins` file and move the style declarations directly into the `security--component` declaration? I think we'll want to move away from `mixins` if we're not using them elsewhere.

Reply:
What do you mean by "if we're not using them else"? Isn't this a pattern that most components use right now?

Reply:
For context - Where a mixin or variable is not required by another module, I'd like to push for us to have a single entry point for components to reduce cognitive overload where necessary, similarly to https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/Accordion/_index.scss or https://github.com/carbon-design-system/ibm-security/blob/dev/src/components/StringFormatter/_index.scss.

I'm fine with this being addressed outside of this PR though.

Reply:
Thanks for the explanation -- I'm fine changing this here & removing the mixins file 👍